yang/frr-vrf.yang

```diff
@@ -0,0 +1,33 @@
+module frr-vrf {
+  yang-version 1.1;
+  namespace "http://frrouting.org/yang/vrf";
+  prefix frr-vrf;
+
+  organization
+    "Free Range Routing";
+  contact
+    "FRR Users List:       <mailto:frog@lists.frrouting.org>
+     FRR Development List: <mailto:dev@lists.frrouting.org>";
+  description
+    "This module defines a model for managing FRR VRFs.";
+
+  revision 2019-12-03 {
+    description
+      "Initial revision.";
+  }
+
+  container lib {
+    list vrf {
+      key "name";
+      description
+        "vrf name.";
+      leaf name {
+        type string {
+          length "1..36";
```

Comment:
1..16, no? And I believe the name of a vrf should be its own type encapsulating this constraint.

Reply:
done
I have raised the new PR for common yang modules.
Please refer to this PR.
https://github.com/FRRouting/frr/pull/5525